src/controllers/api/account.ts

```diff
@@ -381,6 +374,69 @@ export class AccountController {
 		return response.status(StatusCodes.OK).json({});
 	}
 
+	static async provisionCustomerAccount(
+		network: CheqdNetwork,
+		accounts: PaymentAccountEntity[],
+		customerEntity: CustomerEntity
+	): Promise<SafeAPIResponse<PaymentAccountEntity>> {
+		const existingAccount = accounts.find((acc) => acc.namespace === network);
```

Comment:
Do we need to check for mainnet permissions before creating the mainnet account?

Reply:
No @DaevMithran we create testnet and mainnet accounts for all users